finance/api/controllers/DadosController.js

```diff
@@ -0,0 +1,59 @@
+const dataBase = require('../models')
+
+class DadosController{
+    static async pegaTodosOsDados(req, res){
+        try{
+            const pegaTodosOsDados = await dataBase.dbFinance.findAll()
+            return res.status(200).json(pegaTodosOsDados)            
+        }   catch(error){
+            return res.status(500).json(error.message)
+        }
+    }
+    
+    static async pegaUmDado(req, res){
```

Comment:
Seria bom validar ser a entidade não for encontrada e retornar o erro 404 nestes casos.